public/js/views/management.jsx

```diff
@@ -26,6 +29,11 @@ export default class Management extends Component {
     }
   }
 
+  showSubscriptions = click => {
+    click.preventDefault();
```

Comment:
`click` seems a bit unconventional since it's an event object not a click per se.